sdks/nodejs/spec/alphaAgonesSDK.spec.js

```diff
@@ -0,0 +1,250 @@
+// Copyright 2019 Google LLC All Rights Reserved.

```

Comment:
```suggestion
// Copyright 2020 Google LLC All Rights Reserved.
```

2020

Reply:
This is the only thing that still needs changing, and then this good to merge in my opinion :+1: